sled-agent/src/sled_agent.rs

```diff
@@ -231,6 +231,10 @@ impl SledAgent {
         let mut command = std::process::Command::new(PFEXEC);
         let cmd = command.args(&[
             "/usr/sbin/routeadm",
+            // Needed to access Nexus's external interface.
+            "-e",
+            "ipv4-forwarding",
```

Comment:
Is this still required? The IP address is in the nexus zone, but the link is in the global zone. It seems like it should deliver the traffic right to the zone's IP stack, no forwarding or routing required.

Reply:
You're right - I just tested this, and the forwarding is not necessary.